Packs/Armis/Integrations/Armis/Armis.yml

```diff
@@ -0,0 +1,483 @@
+    name: armis-search-alerts-by-aql-string
+    outputs:
+    - contextPath: Armis.Alert.activityIds
+      description: The activity IDs of the alert.
+      type: Number
+    - contextPath: Armis.Alert.activityUUIDs
+      description: The activity UUIDs of the alert.
+      type: String
+    - contextPath: Armis.Alert.alertId
+      description: The ID of the alert.
+      type: Number
+    - contextPath: Armis.Alert.connectionIds
+      description: The connection IDs of the alert.
+      type: Number
+    - contextPath: Armis.Alert.description
+      description: The description of the alert.
+      type: String
+    - contextPath: Armis.Alert.deviceIds
+      description: The device IDs of the alert.
+      type: Number
+    - contextPath: Armis.Alert.severity
+      description: The severity of the alert.
+      type: String
+    - contextPath: Armis.Alert.status
+      description: The status of the alert.
+      type: String
+    - contextPath: Armis.Alert.time
+      description: The date and time the alert occurred.
+      type: Date
+    - contextPath: Armis.Alert.title
+      description: The title of the alert.
+      type: String
+    - contextPath: Armis.Alert.type
+      description: The type of the alert.
+      type: String
+  - arguments:
+    - default: false
+      description: The ID of the device to add a tag to.
+      isArray: false
+      name: device_id
+      required: true
+      secret: false
+    - default: false
+      description: The tags to add to the device.
+      isArray: true
+      name: tags
+      required: true
+      secret: false
+    deprecated: false
+    description: Add a tag to a device.
+    execution: false
+    name: armis-tag-device
+  - arguments:
+    - default: false
+      description: The ID of the device to remove a tag from.
+      isArray: false
+      name: device_id
+      required: true
+      secret: false
+    - default: false
+      description: The tags to remove from the device.
+      isArray: true
+      name: tags
+      required: true
+      secret: false
+    deprecated: false
+    description: Remove a tag from a device.
+    execution: false
+    name: armis-untag-device
+  - arguments:
+    - default: false
+      description: The name of the device to search for.
+      isArray: false
+      name: name
+      required: false
+      secret: false
+    - default: false
+      description: The ID of the device to search for.
+      isArray: false
+      name: device_id
+      required: false
+      secret: false
+    - default: false
+      description: The MAC address of the device to search for.
+      isArray: false
+      name: mac_address
+      required: false
+      secret: false
+    - default: false
+      description: The IP address of the device to search for.
+      isArray: false
+      name: ip_address
+      required: false
+      secret: false
+    - default: false
+      description: The device type to search for.
+      isArray: true
```

Comment:
Please mention this is a comma-separated list of values and add possible values list like done in the risk_level argument